.editorconfig

```
@@ -0,0 +1,28 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[{*.patch,syntax_test_*}]
+trim_trailing_whitespace = false
+
+[{*.c,*.cpp,*.h}]
+charset = utf-8
+
+[{*.c,*.cpp,*.h,Makefile}]
+trim_trailing_whitespace = true
+insert_final_newline = true
+end_of_line = lf
+indent_style = space
+indent_size = 2
+
+[{*.py,*.conf,*.sublime-project}]
+indent_style = tab
+indent_size = 4
```

build.sh clones and pulls down pkg2appimage as well as part of the PrusaSlicer repository. It is unlikely, but these might be frustrating to someone editing the files within to change something (some of the Makefiles for instance, are tab-delimited).

I would suggest removing these (since the top-level utilities don't use them) and adding instead (so that all files in pkg2appimage and below have their own styling preserved):
```suggestion
[/pkg2appimage/**]
indent_style = unset
indent_size = unset
tab_width = unset
end_of_line = unset
charset = unset
trim_trailing_whitespace = unset
insert_final_newline = unset
```